src/librustc/traits/mod.rs

```diff
@@ -484,7 +484,8 @@ EnumTypeFoldableImpl! {
 
 pub struct FulfillmentError<'tcx> {
     pub obligation: PredicateObligation<'tcx>,
-    pub code: FulfillmentErrorCode<'tcx>
+    pub code: FulfillmentErrorCode<'tcx>,
+    pub points_at_arg_span: bool,
```

Comment:
Could use a doc comment explaining what it means. Alternatively / Additionally, consider an `enum` instead to make things more obvious.